### PR TITLE
Pipelines are wrongly mentioned in the Forwarding to Azure Monitor Logs section

### PR DESCRIPTION
**Pull request by prithvipatil97**
- Pipelines are wrongly mentioned in the Advanced configuration options of Forwarding to Azure Monitor Logs sections.

- Here is the documentation link: https://docs.openshift.com/container-platform/4.16/observability/logging/log_collection_forwarding/configuring-log-forwarding.html#logging-forwarding-azure_configuring-log-forwarding

- Here are the current `Advanced configuration options` part:
 ~~~
apiVersion: "logging.openshift.io/v1"
kind: "ClusterLogForwarder"
metadata:
  name: instance
  namespace: openshift-logging
spec:
  outputs:
  - name: azure-monitor
    type: azureMonitor
    azureMonitor:
      customerId: my-customer-id
      logType: my_log_type
      azureResourceId: "/subscriptions/111111111" 
      host: "ods.opinsights.azure.com" 
    secret:
       name: my-secret
  pipelines:
   - name: app-pipeline <<====== Wrong indentation
    inputRefs:                      
    - application                  
    outputRefs:                   
    - azure-monitor
~~~

- Due to this ClusterLogForwarder can not get applied. It streams the following errors:

~~~
Error parsing YAML: YAMLException: bad indentation of a sequence entry at line 19, column 5: 
inputRefs: 

Error parsing YAML: YAMLException: bad indentation of a sequence entry at line 20, column 5:
- application

Error parsing YAML: YAMLException: bad indentation of a sequence entry at line 21, column 5: 
outputRefs:

Error parsing YAML: YAMLException: bad indentation of a sequence entry at line 22, column 5:
- azure-monitor
~~~

- Here is the correct indentation for `Advanced configuration options` part: 
~~~
apiVersion: "logging.openshift.io/v1"
kind: "ClusterLogForwarder"
metadata:
  name: instance
  namespace: openshift-logging
spec:
  outputs:
  - name: azure-monitor
    type: azureMonitor
    azureMonitor:
      customerId: my-customer-id
      logType: my_log_type
      azureResourceId: "/subscriptions/111111111" 
      host: "ods.opinsights.azure.com" 
    secret:
       name: my-secret
  pipelines:
  - name: app-pipeline
    inputRefs:
    - application
    outputRefs:
    - azure-monitor
~~~

Version(s):
 

RHCOP 4.16, RHOCP 4.15, RHOCP 4.14, RHOCP 4.13

Issue:

https://issues.redhat.com/browse/OBSDOCS-1574

Link to docs preview:

https://86391--ocpdocs-pr.netlify.app/openshift-dedicated/latest/observability/logging/log_collection_forwarding/configuring-log-forwarding.html

https://86391--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/log_collection_forwarding/configuring-log-forwarding.html

https://86391--ocpdocs-pr.netlify.app/openshift-rosa/latest/observability/logging/log_collection_forwarding/configuring-log-forwarding.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Comment by prithvipatil97** [comment]
Hello @skrthomas ,
Thank you very much for reviewing this PR and providing the approval.
Could you please also add 'peer-review-done' label to this PR.

I will take QE approval before submitting this PR for merge review.

Regards,
Prithviraj Patil.

**Comment by prithvipatil97** [comment]
Hello Team,
I need QE approval for this change.
All tests have passed and Peer review is also done.
@anpingli  , @kabirbhartiRH  , @QiaolingTang  , It would be really helpful if someone could please take a look and provide QE approval for this change.

Thanks in advance.

File: modules/logging-forwarding-azure.adoc [hunk 1]
```diff
@@ -128,10 +128,10 @@ spec:
        name: my-secret
   pipelines:
    - name: app-pipeline
```
Comment (anpingli) [inline modules/logging-forwarding-azure.adoc:130]: can you align '-' with the '-' in line 120. Remove one pre-space.

File: modules/logging-forwarding-azure.adoc [hunk 2]
```diff
@@ -128,10 +128,10 @@ spec:
...
-    inputRefs:
-    - application
-    outputRefs:
-    - azure-monitor
+     inputRefs:
```
Comment (anpingli) [inline modules/logging-forwarding-azure.adoc:131]: Please revert change from 131 to 134.

File: modules/logging-forwarding-azure.adoc (hunk 1 above)
Comment (prithvipatil97) [reply modules/logging-forwarding-azure.adoc]: Hello @anpingli ,
Thank you very much for this suggestion.
Line number 120 is correct.

I performed the suggested changes in line number 130, removed one `pre-space`

Request you to please have a look.

Regards,
Prithviraj Patil

File: modules/logging-forwarding-azure.adoc (hunk 2 above)
Comment (prithvipatil97) [reply modules/logging-forwarding-azure.adoc]: Hello @anpingli ,
Reverted the suggested changes from 131 to 134.

Kindly take a look and provide QE approval for this change.
Thank you.

**Comment by anpingli** [comment]
@skrthomas  the line 112 should be three '-'.  Not four '-'

**Comment by prithvipatil97** [comment]
> @prithvipatil97 the line 112 should be three '-'. Not four '-'

Hello @anpingli ,
Thanks for your suggestion.
As discussed over Slack, line number 112 required four '----', not three '---'

The changes previously suggested for lines 130 to 134 have been implemented.
Request you please take a look and provide your approval.

Regards,
Prithviraj Patil

**Comment by anpingli** [comment]
LGTM

**Comment by prithvipatil97** [comment]
Hello Team,
Due to some technical error, I am unable to squash the recent commit. Stuck at the `git push` command:

~~~
$ git push origin patch-12 -f

Username for 'https://github.com': prithvipatil97
Password for 'https://prithvipatil97@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/prithvipatil97/openshift-docs.git/'
~~~~

Hence I have created a new PR with the latest changes.
Here is the new PR link: https://github.com/openshift/openshift-docs/pull/86486 

Peer review is done for this PR.
QE also approved this PR.

Kindly take a look on the new PR and let me know if any other details required.

Thank you in advance.


Regards,
Prithviraj Patil

**Pull request closed without merging**